homeassistant/components/openevse/strings.json

```
@@ -0,0 +1,39 @@
+{
+    "config": {
+      "abort": {
+        "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+      },
+      "error": {
+        "communication": "Unable to connect to OpenEVSE charger. Please check the log for details."
+      },
+      "step": {
+        "user": {
+          "data": {
+            "name": "[%key:common::config_flow::data::name%]",
+            "host": "[%key:common::config_flow::data::host%]",
+            "password": "[%key:common::config_flow::data::password%] (optional)",
+            "username": "[%key:common::config_flow::data::username%] (optional)"
+          },
+          "description": "Please enter the connection information of OpenEVSE charger.",
+          "title": "OpenEVSE Setup"
+        }
+      }
+    },
+    "options": {
+      "error": {
+        "communication": "Unable to connect to OpenEVSE charger. Please check the log for details."    
+      },
+      "step": {
+        "init": {
+            "data": {
+                "name": "[%key:common::config_flow::data::name%]",
+                "host": "[%key:common::config_flow::data::host%]",
+                "password": "[%key:common::config_flow::data::password%] (optional)",
+                "username": "[%key:common::config_flow::data::username%] (optional)"
+            },
+            "description": "Please enter the connection information of OpenEVSE charger.",
+            "title": "OpenEVSE Setup"
```

You can make a key reference to the integration strings.
```suggestion
            "description": "[%key:component::openevse::config::step::user::description%]"
```